docs/tfengine/schemas/project.md

```diff
@@ -38,7 +44,7 @@ Type: boolean
 
 ### project.project_id
 
-ID of project to create.
+ID of project to create and/or provision resources.
```

Comment:
Nit suggestion: "to create and/or provisions **in**"

Reply:
Done